forecasttools/trajectories_to_quantiles.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import polars as pl
+
+
+def trajectories_to_quantiles(
+    trajectories: pl.DataFrame,
+    quantiles: list[int] = None,
```

Comment:
the default list of quantiles is not a list of integers. should this be `list[float]`?

Reply:
Mindless error on my end. You are correct.